File: hospital-be-main/src/HospitalLibrary/Core/DTO/ScheduleAppointmentDTO.cs
using HospitalLibrary.Core.Model;
using System;

namespace HospitalLibrary.Core.DTO
{
    public class ScheduleAppointmentDTO
    {
        public Guid DoctorId { get; set; }
        public DateRange Range { get; set; }

        public ScheduleAppointmentDTO() { }

    }
}

File: hospital-fe-main/src/app/modules/hospital/model/scheduleAppointmentDTO.ts
import {DateRange} from "./date-range.model";

enum Status{
  Pending, //kreiran, nije resen
  Canceled, //otkazan
  Finished //zavrsen
}

export class ScheduleAppointmentDTO {
  doctorId: string = "";
  range!: DateRange;


  public constructor(doctor: string = "",  rangee: DateRange) {
      this.doctorId = doctor;
      this.range = rangee;
  }
}
